src/pages/Scheduling/components/CreateScheduleExceptionSheet.tsx

```diff
@@ -60,41 +60,60 @@ export default function CreateScheduleExceptionSheet({
   // Voluntarily masking the setQParams function to merge with other query params if any (since path is not unique within the user availability tab)
   const [qParams, _setQParams] = useQueryParams<QueryParams>();
   const setQParams = (p: QueryParams) => _setQParams(p, { replace: false });
-
   const formSchema = z
     .object({
       reason: z.string().min(1, t("field_required")),
-      valid_from: z.date({ required_error: t("field_required") }),
-      valid_to: z.date({ required_error: t("field_required") }),
+
+      // Ensure valid_from is today or future
+      valid_from: z
+        .date({ required_error: t("field_required") })
+        .refine((date) => dayjs(date).isAfter(dayjs().subtract(1, "day")), {
+          message: t("date_must_be_future"),
+        }),
```

Comment:
why complicate the logic? 🤔 

can't we simply use `.max` instead? 🤔 

refer docs: https://zod.dev/?id=dates-1